apisix/plugins/proxy-rewrite.lua

```diff
@@ -161,7 +166,9 @@ function _M.rewrite(conf, ctx)
     end
 
     local upstream_uri = ctx.var.uri
-    if conf.uri ~= nil then
+    if conf.use_real_request_uri_unsafe then
+        upstream_uri = core.utils.resolve_var("real_request_uri", ctx.var)
```

Comment:
"real_request_uri" is a typo?

Reply:
He originally wrote `$real_request_uri`, but I suggested changing it to `real_request_uri` because `core.utils.resolve_var` doesn't seem to need `$` prefix?

Reply:
Updated to use `ctx.var.real_request_uri`